controller/controller_test.go

```diff
@@ -754,18 +754,17 @@ const (
 	queueCheckTimeout = shortDelay + 500*time.Millisecond
 )
 
-func pollQ(q workqueue.RateLimitingInterface, sig chan struct{}) func() (bool, error) {
+func pollQ(q workqueue.RateLimitingInterface, sig chan int) func() (bool, error) {
 	return func() (bool, error) {
 		if q.Len() > 0 {
-			sig <- struct{}{}
+			sig <- q.Len()
```

Comment:
Fetch q.Len() only once here for consistency?

Reply:
oops, I swear I meant to do that. yep one sec.

Reply:
done